internal/build/phase.go

```diff
@@ -172,8 +180,9 @@ func (p *Phase) Run(ctx context.Context) error {
 		ctx,
 		p.docker,
 		p.ctr.ID,
-		logging.NewPrefixWriter(logging.GetWriterForLevel(p.logger, logging.InfoLevel), p.name),
-		logging.NewPrefixWriter(logging.GetWriterForLevel(p.logger, logging.ErrorLevel), p.name),
+		logging.GetWriterForLevel(p.logger, logging.InfoLevel),
+		logging.GetWriterForLevel(p.logger, logging.ErrorLevel),
```

Comment:
Ah, so no more prefixing of log lines from lifecycle binaries? I recall somewhere that this was discussed, not sure where we landed on it.